slayer_rails.gemspec

```diff
@@ -21,10 +21,11 @@ Gem::Specification.new do |spec|
   spec.executables   = spec.files.grep(%r{^exe/}) { |f| File.basename(f) }
   spec.require_paths = ["lib"]
 
-  spec.add_runtime_dependency('slayer', '~> 0.2')
+  spec.add_dependency 'slayer', '~> 0.2'
```

Comment:
After reviewing a bunch of other gems that extend rails, it looks like this is fine. Most of them include `rails` or some subset of `rails` as dependencies in the `Gemfile`.

I think our best bet here is to  `add_runtime_dependency('rails')`